interview-questions.md

```diff
@@ -33,18 +48,53 @@ Then, research the question to expand on your answer. Even if you feel you have
 
```

Comment:
https://github.com/learn-academy-2022-charlie/Syllabus/blob/main/rails/restful-routes-crud.md

The diagram should help here too. Looks like you are understanding the paths and http verbs